singularity-eos/eos/eos_spiner.hpp

```diff
@@ -873,12 +873,12 @@ inline void SpinerEOSDependsRhoT::fixBulkModulus_() {
       Real DPDR = dPdRho_.interpToReal(lRho, lT);
       Real DPDE = dPdE_.interpToReal(lRho, lT);
       Real DEDR = dEdRho_.interpToReal(lRho, lT);
-      Real DTDE = dTdE_.interpToReal(lRho, lT);
+      Real DPDR_T = DPDR + DPDE * DEDR;
```

Comment:
Provided that 
- `DPDR` is $`\left(\frac{\partial P}{\partial \rho}\right)_e`$
- `DEDR` is $`\left(\frac{\partial e}{\partial \rho}\right)_T`$
- `DPDE` is $`\left(\frac{\partial P}{\partial e}\right)_\rho`$
Then yes I think the math is correct

Reply:
thanks for double checking 😸